src/vscode-bicep/src/deploy/selectParameterFile.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as path from "path";
+import vscode, { Uri } from "vscode";
+
+import {
+  IActionContext,
+  IAzureQuickPickItem,
+} from "@microsoft/vscode-azext-utils";
+
+import { appendToOutputChannel } from "../utils/appendToOutputChannel";
+
+export async function selectParameterFile(
```

Comment:
I think we can move this to `deploy.ts` sicne it's specific to the deployment command.

Reply:
We have several enhancements coming up in this area- https://github.com/Azure/bicep/issues/created_by/bhsubra

But for now, I'm okay to move it to deploy.ts. We can refactor as needed.

https://github.com/Azure/bicep/commit/17afe369211abe2912a4277f9b5f3d89d59ff7df

Reply:
Sounds good.